package.json

```diff
@@ -174,6 +176,7 @@
     "timemachine": "0.3.2",
     "transform-loader": "0.2.4",
     "ts-node": "^10.4.0",
+    "typed-scss-modules": "5.0.0",
```

Comment:
Since we are using webpack i would suggest to switch to something automated like https://github.com/Megaputer/dts-css-modules-loader ?

Reply:
Let me check if it works. In any case i added `typed-scss-modules` to gulp https://github.com/input-output-hk/daedalus/pull/2870/commits/307e76b6358a7eb8cd2e511407b25b21e3e83829

Reply:
@DominikGuzei i've found a few issues with this tool:

1- The lib generated types for keyframes animation:

source/renderer/app/components/staking/delegation-center/DelegationCenterHeader.scss

```
  @keyframes rotating-spinner {
    0% {
      transform: rotate(0deg);
    }
    100% {
      transform: rotate(360deg);
    }
  }
```

```
export const rotating-spinner: string;
```

2-Declaration for `switch` classnames are not valid 

![Screen Shot 2022-02-14 at 14 23 07](https://user-images.githubusercontent.com/2846918/153914455-4eec9120-ed40-4021-9127-7f4ec02cbdb2.png)

Reply:
Hm ok, still not happy with the manual effort this would require 😛